pkg/cover/report.go

```diff
@@ -159,13 +159,19 @@ func (rg *ReportGenerator) generate(w io.Writer, progs []Prog, files map[string]
 		Root: new(templateDir),
 	}
 	for fname, file := range files {
-		if !strings.HasPrefix(fname, rg.buildDir) {
+		var remain string
+		if !filepath.IsAbs(fname) {
+			// If it's a relative path, it is relative to the build directory.
+			fname = filepath.Join(rg.buildDir, fname)
+			remain = filepath.Clean(strings.TrimPrefix(fname, rg.srcDir))
```

Comment:
Wait, we just prepended buildDir and now trying to remove srcDir prefix... I don't understand.

Reply:
I changed it. PTAL